api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -270,10 +270,12 @@ private void ensureOrderTypeIsSet(Order order, OrderContext orderContext) {
 			orderType = getOrderTypeByConcept(order.getConcept());
 		}
 		if (orderType == null && order instanceof DrugOrder) {
-			orderType = Context.getOrderService().getOrderTypeByUuid(OrderType.DRUG_ORDER_TYPE_UUID);
+			List<OrderType> types = Context.getOrderService().getOrderTypesByClassName(DrugOrder.class.getName(), true);
```

Comment:
No need for `Context.getOrderService()` here - just call `getOrderTypesByClassName` directly.  We only need to use that full method when we are needing to ensure AOP interceptors are hit, which we don't in this case.